apps/explorer/src/components/module/module-functions-interaction/ModuleFunction.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Disclosure } from '@headlessui/react';
+
+import type { SuiMoveNormalizedFunction } from '@mysten/sui.js';
+import type { ObjectId } from '@mysten/sui.js/src';
+
+import { ReactComponent as ChevronDownIcon } from '~/assets/SVGIcons/16px/chevron-down.svg';
+
+export type ModuleFunctionProps = {
+    packageId: ObjectId;
+    moduleName: string;
+    functionName: string;
+    functionDetails: SuiMoveNormalizedFunction;
+    defaultOpen?: boolean;
+};
+export function ModuleFunction({
+    defaultOpen,
+    functionName,
+    functionDetails,
+}: ModuleFunctionProps) {
+    return (
+        <div className="rounded-lg bg-sui-grey-40">
+            <Disclosure defaultOpen={defaultOpen}>
+                <Disclosure.Button
+                    as="div"
+                    className="flex flex-nowrap cursor-pointer items-center pt-[15px] pb-[17px] px-[20px]"
```

Comment:
Why all the custom padding values?

Reply:
Actually 20 shouldn't be custom, but for the rest trying to follow the design. I guess I could use `pt-3.5` and `pb-4` 1px should be ok, @mystie711 ?

Reply:
done